tests/test_session.py

```diff
@@ -160,7 +171,7 @@ def callback(sample: Sample):
 def test_session():
     zenoh.init_logger()
     (peer01, peer02) = open_session(["tcp/127.0.0.1:17447"])
-    run_session_qryrep(peer01, peer02)
-    run_session_pubsub(peer01, peer02)
+    # run_session_qryrep(peer01, peer02)
```

Comment:
I guess you forgot to put the tests back

Reply:
Oups 😅